recipes/sdl2_mixer/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Make sure TravisCI can find SDL2
+if [ `uname` == Darwin ]; then
+	export LDFLAGS="${LDFLAGS} -Wl,-rpath,$PREFIX/lib"
+fi
+
+sed -i -- "s|@prefix@|${PREFIX}|g" SDL2_mixer.pc.in
+SMPEG_CONFIG="${PREFIX}/bin/smpeg2-config"
+./configure --disable-dependency-tracking --prefix=${PREFIX}
```

Comment:
There doesn't appear to be a make check command, and trying to invoke it causes the process to stop with an error.